lib/conversation/replies.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+/* eslint-disable no-param-reassign */
+
+const logger = require('winston');
+
+const Command = require('./command');
+const actions = require('./actions');
+const helpers = require('../helpers');
+const stathat = require('../stathat');
+const BotRequest = require('../../app/models/BotRequest');
+
+function customMsgDecorator(fn) {
+  return (args) => {
+    fn(args);
+    return helpers.sendResponse(args.res, 200, args.text);
+  };
+}
+
+function renderEndConversationMsg(fn) {
+  return (type, args) => helpers.renderMessageForMessageType(args.req, type)
+      .then((replyMessage) => {
+        args.text = replyMessage;
```

Comment:
I'm on board with a single `args` object -- it was more a question of potentially defining `replyText` on our `req` object (like we do for our `req.user`, `req.campaign` etc) that we pass to `args`. I'm not a strong advocate of one or the other but was just curious about the choice to define as `args` property vs adding to `req`. Regardless of which object it's defined in--  what do you think about renaming it from `text` to `replyText`